pom.xml

```diff
@@ -1630,6 +1669,31 @@
                         <exclusionPatterns>
                             <exclusionPattern>org/joda/time/.*</exclusionPattern>
                         </exclusionPatterns>
+                        <exclusions>
+                            <!-- getOnlyElement is more readable than the stream analogue -->
+                            <exclusion>com/google/common/collect/Iterables.getOnlyElement:(Ljava/lang/Iterable;)Ljava/lang/Object;</exclusion>
+                            <!-- getLast has lower complexity for array based lists than the stream analogue (O(1) vs O(log(N)) -->
+                            <exclusion>com/google/common/collect/Iterables.getLast:(Ljava/lang/Iterable;)Ljava/lang/Object;</exclusion>
+                            <exclusion>com/google/common/collect/Iterables.getLast:(Ljava/lang/Iterable;Ljava/lang/Object;)Ljava/lang/Object;</exclusion>
+                            <!-- TODO -->
```

Comment:
make it more explicit:

<!-- TODO: requires getting getting to common understaning which of those we want to enable -->

Or sth alike.